bochs/bx_debug/dbg_main.cc

```diff
@@ -2258,6 +2260,26 @@ void bx_dbg_disassemble_current(int which_cpu, int print_time)
   }
 }
 
+void bx_dbg_addlyt(const char* new_layoutpath)
+{
+  memset(layoutpath, 0, sizeof(layoutpath));
+  strncpy(layoutpath, new_layoutpath, sizeof(layoutpath) - 2);
+}
+
+void bx_dbg_remlyt(void)
+{
+  layoutpath[0] = '\0';
+}
+
+void bx_dbg_lyt(void)
+{
+  if ('\0' == layoutpath[0]) {
+    return;
+  }
+
+  bx_nest_infile(layoutpath);
+}
```

Comment:
What will cause the debugger to run the script ?

Reply:
This :D 

I am emulating a source command haha. Its tricky but simple and it works fine. 

You can try it! just clone my master: 

git clone https://github.com/therealdreg/Bochs.git

Reply:
![image](https://user-images.githubusercontent.com/9882181/187249419-1c983192-9d80-4f5a-8bf2-f008de38c0af.png)

Reply:
I think something weird happened with the previous commit, now it's fixed :D